index.js

```diff
@@ -251,10 +253,12 @@ if (Check == "!ma "){
     }
 
 
-console.log(attachment.url);
-
 request(attachment.url, { json: true }, (err, res, body) => {
 if (err) { return console.log(err); }
+
+if (String(attachment.filename).endsWith('.yaml') {

```

Comment:
If attachment is yaml parse it as yaml

Reply:
If code is yaml send it as json

Reply:
i meant it should convert yaml to object if its yaml

Reply:
all gud now, thx

Reply:
why did you copied code when you can just accept pull request

Reply:
I had to edit more